### ISSUE TITLE
Incorrect or extra sensitive "No change in charges" alert when rubbing with two balloons present.

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#421]
While spot checking dragging/rubbing with two balloons in #419, I noticed what seemed like incorrect utterances of "No change in charges..." when rubbing on sweater.

When two balloons are present, I seem to hear "No change in charges" right after I pick up charge on the sweater and get notified of a charge transfer.

It's almost like there is some extra sensitivity to that "No change in charges" alert when two balloons are present.

To re-produce, have two balloons present and rub on sweater.

**Comment by terracoda** [comment]
@jessegreenberg, I'm not sure this is a bug, but I have added the bug label.

**Comment by jessegreenberg** [comment]
Good find @terracoda, looking into it.

**Comment by jessegreenberg** [comment]
This was a timing issue, and should be fixed in the above commit. @terracoda can you please verify by trying to reproduce the issue now?

**Comment by terracoda** [comment]
@jessegreenberg, on it!

**Comment by terracoda** [comment]
Sounds great, @jessegreenberg. I am closing the issue.

**Comment by terracoda** [comment]
Oops

**Comment by terracoda** [comment]
Oops maybe that was premature. I leave the closing to you, @jessegreenberg.

**Comment by jessegreenberg** [comment]
Thanks for verifying @terracoda, not premature at all. Closing.
